### DISCUSSION TITLE
Svelteflow: How to extend <Controls>

### DISCUSSION THREAD

**Discussion by lovetox** [xyflow/xyflow#3788] [Q&A]
Hi,

I found the documentation at https://svelteflow.dev/api-reference/components/control-button

1. my type checker tells me i need to set a "class" attribute, the documentation does not mention this attribute, the example also does not have one.

2. If i follow this example i get an error in the console

`<Controls> received an unexpected slot "default".`

The additional button is not shown, no idea what the error means..

**Comment by lovetox** [comment]
for now my solution is to copy the whole Controls code to my project and adjust it.

**Reply by moklick** [reply]
Thanks for the report. We will add it in the next version :)
